src/libra_py/__init__.py

```diff
@@ -12,6 +12,7 @@
            "workflows",
            "acf",
            "autoconnect",
+           "auxiliary_functions",
```

Comment:
auxiliary to what? the name is not descriptive enough - please make sure to call the files in a more descriptive way, split the content of your current file into more than 1 files, if needed - just make sure the content is grouped according to common purposes